### PR TITLE
updated callbacks for rails 5

### PR DESCRIPTION
**Pull request by prosanelli**

### CODE REVIEW

File: lib/mandrill-rails/web_hook_processor.rb [hunk 1]
```diff
@@ -34,8 +34,8 @@ module Mandrill::Rails::WebHookProcessor
   extend ActiveSupport::Concern
 
   included do
-    skip_before_filter :verify_authenticity_token
-    before_filter :authenticate_mandrill_request!, :only => [:create]
+    skip_before_action :verify_authenticity_token, raise: false
```
Comment (PikachuEXE) [inline lib/mandrill-rails/web_hook_processor.rb]: whats `raise: false` for?

Comment (prosanelli) [inline lib/mandrill-rails/web_hook_processor.rb]: https://github.com/rails/rails/pull/19029

**Comment by PikachuEXE** [comment]
Wondering when will this be merged and released...

**Comment by Fuffi** [comment]
Can you merge this?

**Comment by tardate** [comment]
thanks everyone for bugging me about this! I just merged the various efforts from @prosanelli @shanna @PikachuEXE @Fuffi @spastorino @Flarra on #43 #42  and done some cleanup to get a green ci. It is merged on master but I haven't released a gem yet.

Problem is I don't run mandrill with any rails 5 apps yet, so my confidence this actually works is only as good as the unit-like testing. can someone does us all a favour and test this out for real and give a thumbs up/down?

**Pull request merged**